components/Navigation/Navigation.html

```diff
@@ -42,8 +42,6 @@
         <div class="right-nav-menu">
             <ion-icon name="menu-outline"></ion-icon>
         </div>
-
-
     </nav>
    
     <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
```

Comment:
Place the icon script tags in the head section as icons will also parse with the HTML file.